src/senaite/core/content/sampletemplate.py

```diff
@@ -0,0 +1,635 @@
+            default=u"Preservation"
+        ),
+        allowed_types=("SamplePreservation", ),
+        multi_valued=False,
+        required=False)
+
+    # SAMPLE TYPE
+    directives.widget(
+        "sampletype",
+        UIDReferenceWidgetFactory,
+        catalog=SETUP_CATALOG,
+        query={
+            "portal_type": "SampleType",
+            "is_active": True,
+            "sort_on": "sortable_title",
+            "sort_order": "ascending",
+        },
+        display_template="<a href='${url}'>${Title}</a>",
+        columns=get_default_columns,
+        limit=5)
+    sampletype = UIDReferenceField(
+        title=_(
+            u"label_sampletemplate_partition_sampletype",
+            default=u"Sample Type"
+        ),
+        allowed_types=("SampleType", ),
+        multi_valued=False,
+        required=False)
+
+
+class ISampleTemplateSchema(model.Schema):
+    """Schema interface
+    """
+
+    # Fieldset "Partitions"
+    model.fieldset(
+        "partitions",
+        label=_(u"fieldset_sampletemplate_partitions",
+                default=u"Partitions"),
+        fields=[
+            "partitions",
+            "auto_partition",
+        ]
+    )
+
+    # Fieldset "Analyses"
+    model.fieldset(
+        "analyses",
+        label=_(u"fieldset_sampletemplate_analyses",
+                default=u"Analyses"),
+        fields=[
+            "services",
+        ]
+    )
+
+    # Title
+    title = schema.TextLine(
+        title=_(u"title_sampletemplate_title",
+                default=u"Name"),
+        required=True)
+
+    # Description
+    description = schema.Text(
+        title=_(u"title_sampletemplate_description",
+                default=u"Description"),
+        required=False)
+
+    # Sample Point
+    directives.widget(
+        "samplepoint",
+        UIDReferenceWidgetFactory,
+        catalog=SETUP_CATALOG,
+        query={
+            "portal_type": "SamplePoint",
+            "is_active": True,
+            "sort_on": "sortable_title",
+            "sort_order": "ascending",
+        },
+        display_template="<a href='${url}'>${Title}</a>",
+        columns=get_default_columns,
+        limit=5)
+    samplepoint = UIDReferenceField(
+        title=_(u"label_sampletemplate_samplepoint",
+                default=u"Sample Point"),
+        description=_(u"description_sampletemplate_samplepoint",
+                      default=u"Select the sample point for this template"),
+        allowed_types=("SamplePoint", ),
+        multi_valued=False,
+        required=False)
+
+    # Sample Type
+    directives.widget(
+        "sampletype",
+        UIDReferenceWidgetFactory,
+        catalog=SETUP_CATALOG,
+        query={
+            "portal_type": "SampleType",
+            "is_active": True,
+            "sort_on": "sortable_title",
+            "sort_order": "ascending",
+        },
+        display_template="<a href='${url}'>${Title}</a>",
+        columns=get_default_columns,
+        limit=5)
```

Comment:
I would remove `limit=5`, is already the limit set by default in `QuerySelectWidget`